docs/devenv_rhel8.md

```diff
@@ -0,0 +1,248 @@
+## Create Development Virtual Machine
+Start by downloading the RHEL 8.6 or above ISO image from the https://developers.redhat.com/products/rhel/download location. 
+
+### Creating VM
+Create a RHEL 8.x virtual machine with 2 cores, 4096MB of RAM and 40GB of storage. Move the ISO image to `/var/lib/libvirt/images` directory and run the following command to create a virtual machine.
+```bash
+VMNAME="microshift-dev"
+sudo -b bash -c " \
+cd /var/lib/libvirt/images/ && \
+virt-install \
+    --name ${VMNAME} \
+    --vcpus 2 \
+    --memory 4096 \
+    --disk path=./${VMNAME}.qcow2,size=40 \
+    --network network=default,model=virtio \
+    --os-type generic \
+    --events on_reboot=restart \
+    --cdrom ./rhel-8.6-$(uname -i)-boot.iso \
+"
+```
+
+In the OS installation wizard, set the following options:
+- Root password and `microshift` administrator user
+- In the Installation Destination, select automatic partitioning on the disk without encryption
+- Connect network card and set the hostname (i.e. `microshift-dev.localdomain`)
+- Register the system with Red Hat using your credentials (toggle off Red Hat Insights connection)
+- In the Software Selection, select Minimal Install base environment and toggle on Headless Management to enable Cockpit
+
+Click on Begin Installation and wait until the OS installation is complete.
+
+### Configuring VM
+Log into the virtual machine using SSH with the `microshift` user credentials.
+
+Run the following commands to upgrade the system, install basic dependencies, enable remote Cockpit console and configure SUDO.
+```bash
+sudo dnf update -y
+sudo dnf install -y git cockpit make golang
+sudo systemctl enable --now cockpit.socket
+echo -e 'microshift\tALL=(ALL)\tNOPASSWD: ALL' | sudo tee -a /etc/sudoers
```

Comment:
The diff view isn't great. I'm suggesting 2 changes:

1. Set up sudo before anything else, to avoid typing passwords multiple times.
2. Use a file in `sudoers.d` instead of the global `sudoers` file to avoid collisions if that file changes during an upgrade.

```suggestion
sudo echo -e 'microshift\tALL=(ALL)\tNOPASSWD: ALL' > /etc/sudoers.d/microshift
sudo dnf update -y
sudo dnf install -y git cockpit make golang
sudo systemctl enable --now cockpit.socket
```

Reply:
Done